packages/angular_devkit/build_angular/src/builders/jest/index.ts

```diff
@@ -85,6 +103,7 @@ export default createBuilder(
       jest,
 
       `--rootDir="${path.join(testOut, 'browser')}"`,
+      `--config=${path.join(__dirname, 'jest.config.mjs')}`,
```

Comment:
I assume there is no way to prevent Jest from searching for a config?

Reply:
Not to my knowledge. I tried `--config /dev/null` but Jest rejects that because it doesn't have the right file extension (.js, .ts, .json, etc.)

Eventually we'll probably need to supply our own config anyways, just to customize certain options not available on the CLI, so I don't think there's much benefit in omitting the file altogether.